### ISSUE TITLE
Can't install opscode-reporting

### ISSUE THREAD

**Issue by bighoax** [chef/chef-server#661]
Hello,

I'm running version `chef-server 12.3.1`, and my distribution is ubuntu version 14.04.
I was trying to install `opscode-reporting` but I m receiving this error :

```
/opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/omnibus-ctl-0.4.2/lib/omnibus-ctl.rb:462:in `remove_old_node_state': undefined method `success?' for nil:NilClass (NoMethodError)
	from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/omnibus-ctl-0.4.2/lib/omnibus-ctl.rb:478:in `run_chef'
	from (eval):40:in `block in load_file'
	from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/omnibus-ctl-0.4.2/lib/omnibus-ctl.rb:190:in `call'
	from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/omnibus-ctl-0.4.2/lib/omnibus-ctl.rb:190:in `block in add_command_under_category'
	from /opt/opscode/embedded/lib/ruby/gems/2.1.0/gems/omnibus-ctl-0.4.2/lib/omnibus-ctl.rb:661:in `run'
	from /opt/opscode/embedded/service/omnibus-ctl/chef-server-ctl:217:in `<main>'
 ```

And I used this command:

```
chef-server-ctl install opscode-reporting
```

What am I doing wrong? How can I get past this?

**Comment by marcparadise** [comment]
Are you able to perform a `chef-server-ctl reconfigure` without seeing a similar error?

**Comment by bighoax** [comment]
When I tried to perform a `chef-server-ctl reconfigure` server would become unresponsive. I guess I haven't read the system requirements beforehand, I've spun a very weak instance to try and install chef-server, and it probably errored out during the installation which I might noticed or whatever was the reason. I used the same steps on another machine with slightly better CPU and about 3.75 GB of RAM, works well so far, sorry about the false-alarm.
